app/configs/routes.ts

```diff
@@ -0,0 +1,44 @@
+import {
+  FaShieldHeart,
+  FaGaugeHigh,
+  FaLocationDot,
+  FaUserGroup,
+  FaCloudArrowUp,
+} from "react-icons/fa6";
+import { IconType } from "react-icons";
+
+type Route = {
```

Comment:
still trying to figure out when to use types vs interfaces. from these [typescript docs](https://www.typescriptlang.org/docs/handbook/2/everyday-types.html#differences-between-type-aliases-and-interfaces):

>  "For the most part, you can choose based on personal preference, and TypeScript will tell you if it needs something to be the other kind of declaration. If you would like a heuristic, use interface until you need to use features from type."

Reply:
yes, i think that's a good enough rule. i changed it to an interface🙏